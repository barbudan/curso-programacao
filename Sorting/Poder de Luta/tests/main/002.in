5
Goku 8001
Kuririn 98
Gohan 67
Piccolo 3400
Chaos 67