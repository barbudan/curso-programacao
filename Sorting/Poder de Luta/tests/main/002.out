Goku 8001
Piccolo 3400
Kuririn 98
Chaos 67
Gohan 67
